### PR TITLE
Generated code for the latest public preview version package-2024-03-01

### PR DESCRIPTION
**Pull request by siddiavinashmsft**
…1-preview

# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Review by jsquire — CHANGES_REQUESTED** [review]
@siddiavinashmsft:

Your account lacks the public GitHub Azure organization required of an internal contributor.  Please review the Azure SDK onboarding documentation and use the associated Teams channel for assistance.
- [Azure SDK onboarding](https://eng.ms/docs/products/azure-developer-experience/onboard/requirements) _(Microsoft internal)_
- [Azure SDK onboarding assistance](https://teams.microsoft.com/l/channel/19%3ade3fce22c2994be18cac50502c55f717%40thread.skype/Azure%2520SDK%2520Onboarding?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) _(Microsoft internal)_

Please also be sure to add yourself to [CODEOWNERS](https://github.com/Azure/azure-sdk-for-net/blob/main/.github/CODEOWNERS) for this library, if you will be maintaining it going forward.

**Comment by archerzz** [comment]
@siddiavinashmsft please fix the errors reported in pipeline

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -806,6 +806,9 @@
 # AzureSdkOwners:                                                  @ShivangiReja
 # ServiceOwners:                                                   @arv100kri @bleroy @tjacobhi
 
+# ServiceLabel: %SelfHelp
```
Comment (jsquire) [inline .github/CODEOWNERS:809]: Please move this down to the management section around L957, preserving alphabetical ordering by tags.  You'll need to include both the PR and Service ownership and should follow the existing patterns there.

As there is no existing label, please use "Self Help", which I have created.

**Review by jsquire — DISMISSED** [review]
Please be aware that you are still missing the public `Azure` organization membership required.  This PR cannot move forward until that is corrected.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -951,6 +951,12 @@
 # ServiceLabel: %Recovery Services Backup %Mgmt
 # ServiceOwners:                                                   @Daya-Patil
 
+# PRLabel: %Self Help
+/sdk/selfhelp/Azure.ResourceManager.*/                             @siddiavinashmsft @archerzz @ArcturusZhang @ArthurMa1978
+
+# ServiceLabel: %Self Help
```
Comment (jsquire) [inline .github/CODEOWNERS:957]:
```suggestion
# ServiceLabel: %Self Help %Mgmt
```

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -951,6 +951,12 @@
...
+# ServiceOwners:                                                   @siddiavinashmsft
```
Comment (jsquire) [inline .github/CODEOWNERS:958]: Please fix spacing so that this aligns with the format.

**Review by ArthurMa1978 — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (ArthurMa1978) [reply .github/CODEOWNERS]: fixed

**Pull request merged**